vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -20,7 +20,6 @@ CPU_ARCH=$(getCPUArch)  #amd64 or arm64
 VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
 COMPONENTS_FILEPATH=/opt/azure/components.json
 PERFORMANCE_DATA_FILE=/opt/azure/vhd-build-performance-data.json
-MANIFEST_FILEPATH=/opt/azure/manifest.json
```

Comment:
can we also remove this from the packer templates if we don't need it anymore?

Reply:
Good catch, yes, I've removed that too.